sale_mrp_link/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Alex Comba - Agile Business Group
+# Copyright 2016-2018 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Sale MRP Link',
+    'summary': 'Show manufacturing orders generated from sales order',
+    'version': '10.0.1.0.0',
+    'development_status': 'Beta',
```

Comment:
any prevision to change this someday?

Reply:
could you please remember me what does it involve? Any documentation link would be appreciated!

Anyway, are you fine for merging this PR as is? I'd give a try removing changes related to `.travis.yml` once https://github.com/OCA/sale-workflow/pull/690 will be merged. What do you think?

Reply:
See https://odoo-community.org/page/development-status.

If you would give a try when https://github.com/OCA/sale-workflow/pull/690 gets merged (that would be soon), then let's wait a bit 😉

Reply:
https://github.com/OCA/sale-workflow/pull/690 was merged :)

Reply:
@mreficent, I had a try but unfortunately it doesn't work: https://travis-ci.org/OCA/sale-workflow/builds/426616866
Therefore I'd revert my last commit: https://github.com/OCA/sale-workflow/pull/661/commits/9b8beb798fdeda3e6a4fe29746d7b75aadf85510 Is it ok for you?